src/pages/register.tsx

```diff
@@ -0,0 +1,216 @@
+  const [isSubmit, setIsSubmit] = React.useState<boolean>(false);
+
+  const handleDisplayNameChange = (
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void => {
+    setDisplayName(event.target.value);
+    setIsSubmit(false);
+  };
+
+  function handleEmailChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setEmail(event.target.value);
+    setIsSubmit(false);
+  }
+
+  function handlePasswordChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setPassword(event.target.value);
+    setIsSubmit(false);
+  }
+
+  function handleConfirmPasswordChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setConfirmPassword(event.target.value);
+    setIsSubmit(false);
+  }
+
+  const handleSelectChange = (event: SelectChangeEvent): void => {
+    setGender(event.target.value as string);
+  };
+
+  const displayNameErr: validation = validateTextField(displayName, 1, 100);
+  const emailErr: validation = validateEmail(email);
+  const passwordErr: validation = validatePasswordTextField(
+    password,
+    6,
+    100,
+    isValidConfirmPassword
+  );
+  const confirmPasswordErr: validation = validateConfirmPasswordTextField(
+    confirmPassword,
+    6,
+    100,
+    isValidConfirmPassword
+  );
+
+  const createAccountBtnOnClick = async () => {
+    setIsSubmit(true);
+    setIsValidGender(!(gender === ""));
+    setIsValidBirthDate(!(birthDate === null));
+    setIsValidConfirmPassword(true);
+    if (password.length > 5 && confirmPassword.length > 5) {
+      setIsValidConfirmPassword(password === confirmPassword);
+    }
+    const readyToCreate: boolean = !(
+      displayNameErr.err ||
+      emailErr.err ||
+      passwordErr.err ||
+      confirmPasswordErr.err ||
+      !isValidConfirmPassword
+    );
+    if (readyToCreate) {
+    }
+  };
+
+  const helperText = {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+  };
+
+  return (
+    <Stack spacing={3} alignItems="center" justifyContent="center" style={{ minHeight: "100vh" }}>
+      <Logo width={119} height={119} />
+
+      <Box style={{ width: "50vw" }}>
+        <CommonTextField
+          header="Username"
+          placeholder="Display Name"
+          value={displayName}
+          handleValueChange={handleDisplayNameChange}
+          char_limit={100}
+          isErr={isSubmit && displayNameErr.err}
+          errMsg={displayNameErr.msg}
+        />
+      </Box>
+
+      <Box style={{ width: "50vw", marginTop: 0 }}>
+        <CommonTextField
+          header="Email"
+          placeholder="Email"
+          value={email}
+          handleValueChange={handleEmailChange}
+          isErr={isSubmit && emailErr.err}
+          errMsg={emailErr.msg}
+        />
+      </Box>
+
+      <Box style={{ width: "50vw" }}>
+        <PasswordTextFeild
```

Comment:
as refer to figma, you still missing char count in both password and confirm password

Reply:
Didn't change the design, but what we've talked that it doesn't count anymore.